WordPressKit/RemoteReaderSimplePost.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+struct RemoteReaderSimplePostEnvelope: Decodable {
+    let posts: [RemoteReaderSimplePost]
+}
+
+public struct RemoteReaderSimplePost: Decodable {
+    public enum PostType: Int {
+        case local
+        case global
+        case unknown
+    }
+
+    public let postID: Int
+    public let postUrl: String
+    public let siteID: Int
+    public let isFollowing: Bool
+    public let title: String
+    public let author: RemoteReaderSimplePostAuthor
+    public let excerpt: String
+    public let siteName: String
+    public let featuredImageUrl: String?
+    public let featuredMedia: RemoteReaderSimplePostFeaturedMedia?
+    public let railcar: RemoteReaderSimplePostRailcar
+
+    public var postType: PostType {
+        switch railcar.fetchAlgo {
+        case let algoStr where algoStr.contains("local"):
+            return .local
+        case let algoStr where algoStr.contains("global"):
+            return .global
+        default:
+            return .unknown
+        }
+    }
+
+    private enum CodingKeys: String, CodingKey {
+        case postID = "ID"
```

Comment:
I wish we could reduce the boilerplate but the odd naming of the api fields makes that hard 😞